README.md

```diff
@@ -38,22 +38,20 @@ on the market; it is absolutely free and Open Souce.
+<a href="https://github.com/rem31"><img src="https://avatars.githubusercontent.com/u/73845199?s=400&v=4" title="Rémi Grès" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/jdelpierre"><img src="https://avatars0.githubusercontent.com/u/12846806?s=400&v=4" title="Jérémy Delpierre" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/victorvassilev"><img src="https://avatars1.githubusercontent.com/u/19683390?s=400&v=4" title="Victor Vassilev" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/callapa"><img src="https://avatars1.githubusercontent.com/u/1265083?s=400&v=4" title="Laurent Calvet" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/sc979"><img src="https://avatars1.githubusercontent.com/u/34628915?s=400&v=4" title="Stéphane Chapron" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/vhr"><img src="https://avatars2.githubusercontent.com/u/365020?s=400&v=4" title="Valentin Hristov" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/ganoze"><img src="https://avatars0.githubusercontent.com/u/6575378?s=400&v=4" title="Matthieu Kermagoret" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/loiclau"><img src="https://avatars3.githubusercontent.com/u/2759315?s=400&v=4" title="Loic Laurent" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/leoncx"><img src="https://avatars1.githubusercontent.com/u/3930276?s=400&v=4" title="Maximilien Bersoult" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/jiliangWANG"><img src="https://avatars1.githubusercontent.com/u/48441685?s=400&v=4" title="Jiliang Wang" width="80" height="80"></a> &nbsp;
-<a href="https://github.com/SylvestreG"><img src="https://avatars2.githubusercontent.com/u/3818949?s=400&v=4" title="Sylvestre Gallon" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/Pontissalien"><img src="https://avatars3.githubusercontent.com/u/38663853?s=400&v=4" title="Thomas Arnaud" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/agelibert"><img src="https://avatars1.githubusercontent.com/u/28709386?s=400&v=4" title="Adrien Gélibert" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/Thebarda"><img src="https://avatars3.githubusercontent.com/u/12515407?s=400&u=5642f34672a7c1ea887c6c3e12cd28880aa3971e&v=4" title="Tom Darneix" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/bdauria"><img src="https://avatars0.githubusercontent.com/u/8367233?s=400&u=563bd982e8d1557bdfd39627b029610d9c83fdc6&v=4" title="Bruno D'Auria" width="80" height="80"></a> &nbsp;
 <a href="https://github.com/adr-mo "><img src="https://avatars2.githubusercontent.com/u/31647811?s=400&v=4" title="Adrien Morais" width="80" height="80"></a> &nbsp;
+<a href="https://github.com/jeremyjaouen "><img src="https://avatars.githubusercontent.com/u/61694165?s=400&v=4" title="Jérémy Jaouen" width="80" height="80"></a> &nbsp;
```

Comment:
Can we also add him to the Centreon About pages ?

Reply:
done here https://github.com/centreon/centreon/pull/9521

Reply:
thanks 💯